prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'Jeremy Nguyen';
+      console.log(fullName);
+      console.log(typeof fullName);
+      const numberOfPets = 1;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      const likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      const nothing = null;
+      const never = console.log(nothing);
```

Comment:
you want to use `let` here so you can not define it a value. Const requires a value at declaration and you want a variable with an undefined value here.

Reply:
<img width="483" alt="Screenshot 2023-06-19 at 2 07 35 PM" src="https://github.com/jeremyhnguyen/javascript-prep-0623-code-solutions/assets/136545701/5069aabb-b755-4894-be11-53bbaf49e1a4">